scripts/deploy.js

```diff
@@ -5,36 +5,32 @@ let pathToRepo = require(`path`).resolve(`.`);
 let getStatus = (repo) => repo.getStatus();
 
 let runDeploy = () => {
+  if(process.env.TRAVIS_PULL_REQUEST) {
+    shell.echo(`Skipping deployment for pull request!`);
+    return;
+  }
+
   shell.echo(`Running deployment now...`);
 
-  shell.exec(`git checkout master`);
+  shell.exec(`git stash`);
```

Comment:
This is mainly to clean up anything that might be created by npm test or any other scripts we decide to add later on. It's mainly a precaution but we can remove